tests/pkg_cn-cbor/Makefile

```diff
@@ -0,0 +1,30 @@
+include ../Makefile.tests_common
+
+BOARD_BLACKLIST :=  arduino-duemilanove \
+                    arduino-mega2560 \
+                    arduino-uno \
+                    chronos \
+                    jiminy-mega256rfr2 \
+                    mega-xplained \
+                    msb-430 \
+                    msb-430h \
+                    telosb \
+                    waspmote-pro \
+                    wsn430-v1_3b \
+                    wsn430-v1_4 \
+                    z1 \
+                    #
```

Comment:
I don't understand this lonely comment either.

Reply:
its done in other tests, too see https://github.com/RIOT-OS/RIOT/blob/master/tests/unittests/Makefile#L192 - but its not consistent. I thought it makes it easier to add  a board to the list - simply by putting `<board> \`

Reply:
What is the reason for the `BLACKLIST` here ?
If it is a size issue they should just be put as `BOARD_INSUFFICIENT_MEMORY`.

Reply:
Ok they fail with string.h not found. Tested compiling arduino-mega2560 and wsn430-v1_3b.